### PR TITLE
[Ready] [Modular] Shameless Lizard Buff (Mining Change)

### PR DESCRIPTION
**Pull request by lyricalpaws**
## About The Pull Request
Adds a new crusher trophy, the demonic core. Drops from Demonic Watchers on icemoon, and on mark detonation resets your temperature.

## Why It's Good For The Game
Lizards like to mine too! Also demonic watchers had very little reason to be killed before and just existed to freeze miners and run off.

## Changelog
:cl:
add: Demonic core
/:cl:

## Credits
Halcyon#7104 (Trophy Sprite)

### CODE REVIEW

**Comment by SpaceVampire** [comment]
Unless I am reading this incorrectly, it seems like they did drop the ice wing trophy before?
I assume that would have been a valid reason to kill them, along with the bluespace crystals they drop.
Plus the chance to drop a raw bluespace core on top of that.

**Comment by softcerv** [comment]
> 
> 
> Unless I am reading this incorrectly, it seems like they did drop the ice wing trophy before?
> I assume that would have been a valid reason to kill them, along with the bluespace crystals they drop.
> Plus the chance to drop a raw bluespace core on top of that.

Maybe, I found the original(?) in the code but like it's only attribute is like a bonus damage value of 7 or something and I've never personally gotten one to drop while mining. 

This definitely seems like a good change that will make the demonic ice watchers much less of an annoyance for miners.

**Comment by lyricalpaws** [comment]
On further testing it seems that they dont want to drop it due to their weird way of handling death. I'll try and implement a fix shortly.

**Comment by lyricalpaws** [comment]
Taking off ready til I can fix this shitcode

File: code/modules/mob/living/simple_animal/hostile/mining_mobs/ice demon.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	pull_force = MOVE_FORCE_VERY_STRONG
 	del_on_death = TRUE
 	loot = list()
-	crusher_loot = /obj/item/crusher_trophy/watcher_wing/ice_wing
+	crusher_loot = list(/obj/item/crusher_trophy/demon_core = 1)
```
Comment (GoldenAlpharex) [inline code/modules/mob/living/simple_animal/hostile/mining_mobs/ice demon.dm]: This is what I was trying to tell you over Discord.
```suggestion
	crusher_loot = list(/obj/item/crusher_trophy/demon_core = 1) /// SKYRAT EDIT CHANGE - ORIGINAL : crusher_loot = /obj/item/crusher_trophy/watcher_wing/ice_wing
```

**Comment by lyricalpaws** [comment]
This is ready to be merged when a maintainer has time

**Pull request merged**